tests/default/_core/search/query/terms_set.yaml

```
@@ -0,0 +1,66 @@
+$schema: ../../../../../json_schemas/test_story.schema.yaml
+
+description: Test TermsSetQuery functionality with complex example using movies.
+version: '>= 1.2'
+
+prologues:
+  - path: /movies
+    method: PUT
+    request:
+      payload:
+        mappings:
+          properties:
+            title:
+              type: keyword
+            genres:
+              type: keyword
+            min_required_genres:
+              type: integer
+
+  - path: /movies/_doc/1
+    method: POST
+    parameters:
+      refresh: true
+    request:
+      payload:
+        title: The Lion King
+        genres: [Adventure,Animation, Family]
```

```suggestion
        genres: [Adventure, Animation, Family]
```

I am surprised the linter isn't catching it. You can also put them all on their own lines if you wish.